src/pages/Detail/Detail.js

```diff
@@ -1,8 +1,166 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+// import { useParams } from 'react-router-dom';
 import './Detail.scss';
 
 const Detail = () => {
-  return <div>Detail</div>;
+  const [itemInfo, setItemInfo] = useState({});
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const slides = itemInfo.thumbnail_img;
+  const navigate = useNavigate();
+
```

Comment:
```suggestion
  const navigate = useNavigate();
  
  const slides = itemInfo.thumbnail_img;

```

이렇게 위치 조정되야 어울리겠고,
변수, hooks 사이에 줄바꿈 해줘서 가독성 좋게 만들 수 있습니다